pyuvsim/analyticbeam.py

```diff
@@ -5,9 +5,26 @@
 from __future__ import absolute_import, division, print_function
 
 import numpy as np
+import warnings
 from scipy.special import j1
 
 
+def diameter_to_sigma(diam, freqs):
```

Comment:
I would make sure to explicitly state that "sigma" here is in units of the polar angle rather than units of "angle cosines" which I believe it is often defined by 

https://en.wikipedia.org/wiki/Airy_disk#Approximation_using_a_Gaussian_profile

You may even want to provide an option for both.